NitroxServer-Subnautica/Program.cs

```diff
@@ -32,7 +33,7 @@ public class Program
         private static async Task Main(string[] args)
         {
             // The thread that writers to console is paused while selecting text in console. So console writer needs to be async.
-            Log.Setup(asyncConsoleWriter: true, isConsoleApp: true);
+            Log.Setup(true, isConsoleApp: true);
```

Comment:
Why removing just one side? All or nothing :P

Reply:
It formatted this automatically.